k8s/helm/yona-server/templates/10_deployment_app.yaml

```diff
@@ -38,17 +38,11 @@ spec:
               value: "jdbc:mariadb://{{ .Release.Name }}-mariadb.{{ .Release.Namespace }}.svc.cluster.local/{{ .Values.mariadb.mariadbDatabase }}"
             {{- end }}
             - name: YONA_ANALYSIS_SERVICE_SERVICE_URL
-            {{- if .Values.analysis }}
+            {{- if hasKey .Values.analysis "url_override" }}
               value: {{ .Values.analysis.url_override | quote }}
             {{- else }}
               value: "http://analysis.{{ .Release.Namespace }}.svc.cluster.local"
             {{- end }}
-            - name: YONA_BATCH_SERVICE_SERVICE_URL
```

Comment:
Bad merge, and part of the "edge cases" I was working on.    I have added the full one back and tested against prod and stock values.   I will still have to test with the hasKey handling for edge cases on missing values.yaml items.